docs/Materials.md.html

```diff
@@ -1610,6 +1610,7 @@
 **getWorldGeometricNormalVector()** | float3   |  Normalized normal in world space, before bump mapping (can be used before `prepareMaterial()`)
 **getWorldReflectedVector()**       | float3   |  Reflection of the view vector about the normal (must be used after `prepareMaterial()`)
 **getNdotV()**                      | float    |  The result of `dot(normal, view)`, always strictly greater than 0 (must be used after `prepareMaterial()`)
+**sampleRenderTarget2D(sampler2D, vec2)** | float4 | Similar to the GLSL built-in `texture` but for offscreen render targets
```

Comment:
Could we match the `texture()` name instead? Maybe `renderTarget()`? Also can you not put it in the middle of the get methods in this list?

Reply:
Ah and you should also align the table separators